test/unit/enrichment/schemas/invoice-line.test.js

```diff
@@ -104,4 +104,14 @@ describe('invoice line schema', () => {
     invoiceLine.convergence = false
     expect(schema.validate(invoiceLine)).toBeTruthy()
   })
+
+  test('should fail validation if delivery body is missing', () => {
+    delete invoiceLine.deliveryBody
+    expect(schema.validate(invoiceLine).error).toBeDefined()
+  })
+
+  test('should fail validation if delivery body is incorrect format', () => {
```

Comment:
Could we change the name of this test to be consistent with the other test file? - 'should fail validation if delivery body is not a valid format'